test/organization/complete_ownership_transfer.js

```diff
@@ -0,0 +1,119 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+const Utils = require('../test_lib/utils.js'),
+  { AccountProvider } = require('../test_lib/utils.js'),
+  { Event } = require('../test_lib/event_decoder');
+
+const Organization = artifacts.require('Organization');
+
+contract('Organization::initiateOwnershipTransfer', async (accounts) => {
+
+  describe('Negative Tests', async () => {
+    const accountProvider = new AccountProvider(accounts),
+      owner = accountProvider.get(),
+      proposedOwner = accountProvider.get();
+    let organization = null;
+
+    beforeEach(async function () {
+      organization = await Organization.new({ from: owner });
+      await organization.initiateOwnershipTransfer(
+        proposedOwner,
+        { from: owner },
+      )
+    });
+
+    it('Reverts when caller is not owner.', async () => {
+
+      await Utils.expectRevert(
+        organization.completeOwnershipTransfer({ from: accountProvider.get() },),
+        'Should revert as caller is not proposedOwner.',
+        'Caller is not proposed owner address.',
+      );
+    });
+
+  });
+
+  describe('Storage Tests', async () => {
+
+    const accountProvider = new AccountProvider(accounts),
+      owner = accountProvider.get(),
+      proposedOwner = accountProvider.get();
+    let organization = null;
+
+    beforeEach(async function () {
+      organization = await Organization.new({ from: owner });
+      await organization.initiateOwnershipTransfer(
+        proposedOwner,
+        { from: owner },
+      )
+    });
+
+    it('Should pass when caller is proposed owner.', async () => {
+      assert.ok(
+        await organization.completeOwnershipTransfer(
+          { from: proposedOwner },
+        )
+      );
+
+      assert.strictEqual(await organization.owner.call(), proposedOwner);
```

Comment:
Please confirm that proposedOwner is null.

Reply:
I didn't get the above sentence.
If it's a question then "yes proposedOwner can be null".

Reply:
The function changes the state of `proposedOwner` from whatever to null, so the test should also cover this change of state.